src/main/java/by/onliner/page/components/ShoppingCartPopup.java

```diff
@@ -24,7 +26,12 @@ public void waitForComponentOpened() {
      * @return visibility of popup
      */
     public boolean isShoppingCartPopupVisible() {
-        return shoppingCartPopupCloseButton.isDisplayed();
+       try {
+           Waiter.waitForVisibility(shoppingCartPopupCloseButton, 2, 500);
+           return true;
```

Comment:
Why do you return true not shoppingCartPopupCloseButton.isVisible() ?

Reply:
Fixed